packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -179,6 +183,8 @@ export class DetailsHeaderBase extends BaseComponent<IDetailsHeaderBaseProps, ID
     const isCheckboxHidden = selectAllVisibility === SelectAllVisibility.hidden;
 
     if (!this._dragDropHelper && columnReorderProps) {
+      // TODO Do not assign local fields during render.
+      // This behavior needs to be moved to the appropriate React lifecycle methods.

```

Comment:
Is there an issue open to track the todos in this PR?

Reply:
Added.